demo/grid-filtering-demos.html

```diff
@@ -81,9 +81,16 @@ <h3>Filtering with Data Provider</h3>
                 grid.size = response.size;
 
                 callback(response.result);
+                grid.scrollToIndex(0);
```

Comment:
This causes grid to jump back to top when another request is completed while scrolling:

![jump](https://user-images.githubusercontent.com/10589913/87516512-2eb8c600-c686-11ea-9d19-05bc160ca886.gif)